docs/visual-testing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Sauce Labs Visual Testing
+sidebar_label: Overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Get clear and instant visibility into the impact of code changes on your UI across browsers, devices, and operating systems with our automated visual testing solution, which generates side-by-side comparisons of your UI versions.
+
+Know exactly where and what has changed so that you can deliver a consistent and intuitive user experience.
+
+## Workflows
+
+Sauce Visual testing consists of two main workflows: Test execution and review.
+Both workflows need to implemented, but they may performed by different people or teams.
+Read below to learn more about the workflows.
+
+### Test Execution
+
+This workflow is responsible for generating snapshots (screenshots) of the website or app under test.
+Users new to visual testing often have an existing test suite in WebdriverIO, Cypress or a similar framework.
+With Sauce Visual testing existing test suites can be enhanced with a few extra "take a snapshot" statements and that is all it takes to implement this workflow.
+
+[Follow me to learn more](./visual-testing/workflows/diff-generation.md)
+
+### Review
+
+This workflow is responsible for approving or rejecting detected changes between a snapshot and a baseline.
+Users will be available to review changes and make the baseline evolve for approved changes.
+
+[Follow me to learn more](./visual-testing/workflows/review.md)
+
+
+## Concepts
+
+Sauce Visual Testing comes with its set of concepts that will be defined here.
+
+### Definitions
+
+A **snapshot** is an image that will be compared by Visual Testing. For example: the screenshot of your website or mobile application.
+
+A **build** is a grouping of multiple snapshots in a same container.<br />
```

Comment:
A **build** groups multiple snapshots. E.g. to be able to view all snapshots taken during the execution of your test suite in CI or locally.